### ISSUE TITLE
Upgrading Fauxton within 2.0.0 image

### ISSUE THREAD

**Issue by seantanly** [apache/couchdb-docker#19]
The current Fauxton that is packaged in the 2.0.0 image cannot work correctly with database names that contain forward slash. I'm trying to upgrade the Fauxton within the image, as the newer versions seems to have fixed the problem.

Should the image facilitate the upgrading of Fauxton out of the box? Given there are many outstanding issues in Fauxton that gets resolved and versions released at a pace much faster than the release cycle of CouchDB.

If not, I would appreciate any suggestion on how to upgrade Fauxton within the image manually for the time being. Thanks!

**Comment by garrensmith** [comment]
You could change the tag version in the rebar config for Fauxton to use the master branch. Or you could use the npm version of Fauxton -> `npm install -g Fauxton`

**Comment by seantanly** [comment]
@garrensmith Thank you for the pointer, I opted to upgrade the prepackaged Fauxton within the image instead of running a separate Fauxton container. 

I managed to do so by commenting out the purging of build deps & node_modules folder, and added `git` in `apt-get install` for `npm install -g fauxton` to work. Then I proceeded to replace the prepackaged Fauxton by doing

```
rm -rf /opt/couchdb/share/www
cp -Rp /usr/lib/node_modules/fauxton/dist/release /opt/couchdb/share/www
```

This way, I can still access Fauxton using the original `/_utils` path without further configuration. The drawbacks is the image is much larger compared to the original because the build deps are kept around to allow upgrading of Fauxton when needed.
